Sources/NIOHTTP1Server/main.swift

```diff
@@ -33,6 +33,42 @@ extension String {
     }
 }
 
+private final class HTTPHelloWorldHandler: ChannelInboundHandler {
```

Comment:
@Lukasa @weissi I will remove this before we merge... This was just used for benchmarking